tests/netperf_test.py

```diff
@@ -65,24 +70,37 @@ def run_netperf_parse(self, device, timeout=60):
     def run_netperf(self, device, ip, opts="", timeout=60):
         self.run_netperf_cmd(device, ip, opts=opts)
         return self.run_netperf_parse(device)
+   
+    def kill_netserver(self, device):
+        device.sendline("kill -9 `pidof netserver`")
+        device.expect(prompt)
```

Comment:
This looks like the main missing parts for newer versions of netperf... Looks good.